### DISCUSSION TITLE
Error 500 after Orchard install process

### DISCUSSION THREAD

**Discussion by AlexeyGorbachev** [OrchardCMS/OrchardCore#13216] [Q&A] [answered]
Hi all! I try to install Orchard Core from publish folder to my hosting. I've tried different publish options for publishing (self-contained or framework- dependent)  but I got error 500  or HTTP Error 500.32 (in case of self-contained x64 target runtime) on my site. What is it possible to do? 
 .

**Comment by Piedone** [comment, marked as answer]
We'd need some more info to help you. You can find the Orchard logs under _App_Data/logs_; perhaps those contain a hint. Alternatively, you can temporarily put the app into Development mode ([see the docs](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-7.0)) to see the full error on the error page.

**Reply by AlexeyGorbachev** [reply]
2023-02-09 14:55:15.8544|None|00-b11a89f9ccde948a95963672b36ea4c7-6423fb3e504857e7-00|1|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|ERROR|An unhandled exception has occurred while executing the request. Npgsql.NpgsqlException (0x80004005): Failed to connect to 127.0.0.1:5434 Orchard is trying to connect to my development settings instead of asking me after publishing on new site. Where (what file) in publish folder is it possible to change DB connection data?

**Reply by Piedone** [reply]
By default, check out the tenant-specific appsettings.json under _App_Data/Sites_ in one of the subfolders. Most possibly you only have _Default_.

**Reply by AlexeyGorbachev** [reply]
Thank you. I changed DB settings there to my hosting DB. But Now I see installation page and after input of site, superuser, etc. I got message "The provided database, table prefix and schema are already in use." Where can I see and clear information about database, table prefix and schema, which are already in use?

**Reply by Piedone** [reply]
Have you set up this site before? To clarify:

- The above Postgres exception hints that you've already set this site up with a DB, so changing the connstring in the mentioned file makes sense. However, this is only suitable if you also copied the DB you previously uses for this setup.
- If you want to start anew after deploying the app, then don't change the connstring but delete the whole App_Data folder. Then, you'll see the setup screen, where you can specify all the DB details.

**Reply by AlexeyGorbachev** [reply]
It works! Thank you very much!

**Reply by Piedone** [reply]
Glad to hear!
